src/components/support/Product.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="product">
+    <h2>{{title}}</h2>
```

Comment:
As above, change to `h4` and styling will be done for you
Also add spaces for readability `{{ title }}`